### ISSUE TITLE
Unsupervised Domain Adaptation

### ISSUE THREAD

**Issue by rik030** [phizaz/diffae#37]
Can this method be used for unsupervised domain adaptation?

**Comment by phizaz** [comment]
In what sense of domain adaptation on you which to apply DiffAE, mind providing more specific details? 
One thing I can say though. DiffAE might have some hope on unsupervised image translation problem, e.g. transforming male to female or a cat to a dog and etc. To what extent DiffAE is applicable is a bit underexplored in that regard.

**Comment by rik030** [comment]
Yes, I was asking about the unsupervised image-to-image translation. To be more specific, translating ceT1 modality to hrT2 modality. These modalities are unpaired. So, can DiffAE be used for an unpaired image-to-image translation?

**Comment by phizaz** [comment]
They seem to be medical images, e.g. from https://www.researchgate.net/publication/359141212_Automated_Koos_Classification_of_Vestibular_Schwannoma/figures?lo=1.
![image](https://user-images.githubusercontent.com/451667/202881160-fdf9e4a4-8a3c-4de3-bc51-1aba7c54b368.png)

We have even less clue on medical images. Do both modalities share mostly the same structure? Like cropped the same way? Are the differences between domains are spatially located on the same correspoding regions?

My feeling tells me that you have a good chance on this task. Good luck!

If you have any success, mind you keep we keep in touch? I'm curious how well could in work on medical image domains.
You can mail me directly at konpatp (at) gmail.com

**Comment by rik030** [comment]
Yes, sir, the two modalities share an almost identical structure.
I am going through your paper and the methodology used in your code to check whether domain translation on medical images can be possible with diffae.
Sure, I will be in touch with you :)

**Comment by han-yeol** [comment]
well, I am interested in same object
<img width="971" alt="스크린샷 2022-12-15 오후 3 12 50" src="https://user-images.githubusercontent.com/37497231/207786221-88a6af9a-6c94-4f1a-a522-bc5fc6a6869d.png">

I try to 
T1w mri image (2nd column) to T2w mri image (1st column) image translation 
all model trained mixed T1w and T2w mri image .
and given xT = T1w, cond = T2w slice (similar slice)
but prediction is not good... any advice?

**Comment by han-yeol** [comment]
<img width="1112" alt="스크린샷 2022-12-15 오후 5 19 00" src="https://user-images.githubusercontent.com/37497231/207808328-e49c30b6-7d2e-49a4-bd60-12027a4ee617.png">

This is my best, honestly, This is cherry-picking...

**Comment by phizaz** [comment]
Without the domain (medical) knowledge, I am not sure what's the problem with the prediction you have?
